models/lido/ethereum/lido_ethereum_accounting_deposits.sql

```diff
@@ -0,0 +1,21 @@
+{{ config(
+        alias ='deposits',
+        partition_by = ['period'],
+        materialized = 'table',
+        file_format = 'delta',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "project",
+                                "lido",
+                                \'["pipistrella", "adcv", "zergil1397", "lido"]\') }}'
+        )
+}}
+--https://dune.com/queries/2011901
+--ref{{'lido_ethereum_accounting_deposits'}} 
+
+	SELECT 
+        block_time AS period,
+        amount_staked*POWER(10,18) AS amount_staked,
+        LOWER('0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2') AS token, --ETH
+        tx_hash
+    FROM {{ ref('staking_ethereum_deposits') }} 
```

Comment:
Hi @gregshestakovlido,
since `staking_ethereum_deposits` is an incremental view,
the result may not be static(results may be changed unexpectedly).
If the accounting is linked to certain timeframe, we may hardcoded the time filter in `WHERE` condition.

Reply:
Got it, not sure we could link it to certain timeframe, but will think of how we could avoid this, will return in 1-2 days, thank you

Reply:
The raw table updates regularly as well.
If there is any new deposit between these two addresses, this table will update as well.
And `sum(amount_staked)` will increase over time.

This will result in failure in test, as the test may be run periodically: 
https://github.com/duneanalytics/spellbook/pull/3128/files#diff-47eec97d268bb1c56cfd27f6f77e57dc3e1641b1ab6952f6e08ab38ea6be00ceR3

Reply:
Sorry, I suppose I didn’t get the mistake correctly. In test, we check certain transaction hash, and I thought that within 1 transaction info couldn't be changed (so we can use it for test), but, apparently, am wrong here and data regarding certain transaction could be changed?

Reply:
Sorry I missed the tx_hash part in the where condition.